packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -527,7 +537,7 @@ const New = decorate([
         { saveSchedule },
         storedSchedule = DEFAULT_SCHEDULE
       ) => async (
-        { copyMode, exportMode, snapshotMode },
+        { isDelta, copyMode, exportMode, settings, snapshotMode },
```

Comment:
There is two settings, settings which come from the state (modified by the user) and settings which come from the BD on the backup edition.

`settings` -> `settings = propSettings` to handle the case of the backup edition.